parameter_expansion/pe.py

```diff
@@ -12,33 +12,89 @@
 
 (Pull requests to remove limitations are welcome.)
 
-- Only simple nested expansions of the forms $variable and ${variable} are
-supported and not complex expansions such as in `${foo:-${bar:-$baz}}`
+- Most nested expression expansions should be supported now, but YMMV. Things such
```

Comment:
I think we're beyond 'minimal Bash extensions' at this point -- maybe time to remove the word 'minimal' on line 6.

Reply:
Fixed.